core/drivers/pl061_gpio.c

```diff
@@ -6,6 +6,8 @@
 #include <assert.h>
 #include <drivers/pl061_gpio.h>
 #include <io.h>
+#include <mm/core_memprot.h>
```

Comment:
Why are these added? For the `KEEP_PAGER()` macro you need to add  `<keep.h>` though.

Reply:
Fixed